bika/lims/setuphandlers.py

```diff
@@ -531,41 +530,21 @@ def addColumn(cat, col):
         addIndex(bc, 'cancellation_state', 'FieldIndex')
         addIndex(bc, 'Identifiers', 'KeywordIndex')
 
-        addIndex(bc, 'getDepartmentUIDs', 'KeywordIndex')
         addIndex(bc, 'getAnalysisService', 'KeywordIndex')
         addIndex(bc, 'getAnalyst', 'FieldIndex')
         addIndex(bc, 'getAnalysts', 'KeywordIndex')
-        addIndex(bc, 'getAnalysesUIDs', 'KeywordIndex')
         addIndex(bc, 'BatchDate', 'DateIndex')
-        addIndex(bc, 'getClientOrderNumber', 'FieldIndex')
-        addIndex(bc, 'getClientReference', 'FieldIndex')
-        addIndex(bc, 'getClientSampleID', 'FieldIndex')
-        addIndex(bc, 'getClientTitle', 'FieldIndex')
-        addIndex(bc, 'getClientUID', 'FieldIndex')
         addIndex(bc, 'getContactTitle', 'FieldIndex')
         addIndex(bc, 'getDateDisposed', 'DateIndex')
         addIndex(bc, 'getDateExpired', 'DateIndex')
         addIndex(bc, 'getDateOpened', 'DateIndex')
-        addIndex(bc, 'getDatePublished', 'DateIndex')
         addIndex(bc, 'getDateReceived', 'DateIndex')
         addIndex(bc, 'getDateSampled', 'DateIndex')
```

Comment:
Is `getDateSampled` still necessary? I think this index was only used for Sample objects

Reply:
The view of ```ReferenceSample``` (_bika/lims/browser/referencesample.py_), which is is still indexed in bika catalog makes use of the index ```getDateSampled``` as shown in the snippet below. 

https://github.com/naralabs/bika.lims/blob/23f8e5152aade44996c1d782d43542d484cab820/bika/lims/browser/referencesample.py#L341-L344

This is why ```ReferenceSample``` stil has a DateTimeField called ```DateSampled```

https://github.com/naralabs/bika.lims/blob/23f8e5152aade44996c1d782d43542d484cab820/bika/lims/content/referencesample.py#L97-L101

which leads me to think that the index ```getDateSampled``` is still necessary.